etc/collectd.d/statsd.conf

```diff
@@ -0,0 +1,30 @@
+LoadPlugin statsd
+
+<Plugin statsd>
+  Host "127.0.0.1"
+  Port "8125"
```

Comment:
Let's add commented out instructions for overriding the host/port (see, e.g., [templates/memcached.conf.jinja](https://github.com/Stackdriver/stackdriver-agent-service-configs/blob/master/templates/memcached.conf.jinja)).